packages/element-templates-json-schema-shared/src/defs/base-properties.json

```diff
@@ -26,6 +26,31 @@
           ],
           "errorMessage": "must provide choices=[] with \"Dropdown\" type"
         }
+      },
+      {
+        "if": {
+          "not": {
```

Comment:
Did you try to

* if not: feel is const null
* then: type has to be in enum [String, Text]

Reply:
I could double-check this on Monday 🙂